src/main/java/hudson/plugins/audit_trail/AbstractLogFileAuditLogger.java

```diff
@@ -4,7 +4,7 @@
 import hudson.EnvVars;
 import hudson.Util;
 
-import javax.annotation.Nonnull;
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
Here and elsewhere: adopting the standard convention for Jenkins plugins, which is to use SpotBugs annotations rather than the dead JSR 305 annotations.